File: employeeRecognition/Models/award.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace employeeRecognition.Models
{
    public class award
    {
        public int id { get; set; }
        public int sender_user_id { get; set; }
        public int recipient_user_id { get; set; }
        public string sfn { get; set; }
        public string sln { get; set; }
        public string rfn { get; set; }
        public string rln { get; set; }
        public string type { get; set; }
        public string time { get; set; }
        public string date { get; set; }
    }
}

File: employeeRecognition/ClientApp/src/components/Awards/Award.js
import React from "react";
import { TableAwards } from '../../Shared/TableAwards';

const Award = props => {
    return (
        <tr key={props.awardInfo.id}>
            <td>{props.awardInfo.sfn} {props.awardInfo.sln}</td>
            <td>{props.awardInfo.rfn} {props.awardInfo.rln}</td>
            <td>{props.awardInfo.type}</td>
            <td>{props.awardInfo.time}</td>
            <td>{props.awardInfo.date}</td>
            <TableAwards
                handleDelete={props.handleDelete}
                handleEdit={props.handleEdit}
                awardInfo={props.awardInfo}
            />
        </tr>
    );
}

export { Award };